webapp/channels/jest.config.js

```diff
@@ -34,7 +34,7 @@ const config = {
         '^.*i18n.*\\.(json)$': '<rootDir>/src/tests/i18n_mock.json',
         '^bundle-loader\\?lazy\\!(.*)$': '$1',
     },
-    moduleDirectories: ['', 'node_modules'],
+    moduleDirectories: ['src', 'node_modules'],
```

Comment:
This fixed a bizarre package resolution issue with Jest that I talked about in https://community.mattermost.com/core/pl/mjw31zr1a7n3xnsun65tygwddc